images/wkdev_sdk/Containerfile

```diff
@@ -173,5 +173,29 @@ RUN pkg-config --modversion rice-io && \
 # when deploying the wkdev-sdk image as bot).
 RUN find /etc/systemd/ -type l -name '*.service' -print -exec rm {} \;
 
+### Tools related to webkit-container-infrastructure
+
+# Install OpenTofu
+RUN curl --proto '=https' --tlsv1.2 -fsSL https://get.opentofu.org/install-opentofu.sh -o install-opentofu.sh && \
+    chmod +x install-opentofu.sh && \
+    ./install-opentofu.sh --install-method deb
+
+# Install Helm
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && \
+    chmod +x get_helm.sh && \
+    ./get_helm.sh
+
+# Install packer
+RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - && \
+    apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" && \
+    ${APT_UPDATE} && \
```

Comment:
I don't like repeated apt updates, this means different layers will have slightly different views of packages. The leftovers are also not removed afterwards.